strings/accent_universal.json

```
@@ -44,6 +39,7 @@
 		"satan": "zizo",
 		"adonai": "psydon",
 		"praise psydon": "Mourn the Weeper"
+		" cuck ": "cornuto",
```

```suggestion
		"cuck": "wittol",
```
Period-appropiate word (c. 1500's)